app/src/main/java/com/android/sample/ui/utils/RecipeList.kt

```diff
@@ -258,25 +268,117 @@ fun TopCornerEditButton(recipe: Recipe, onEditClicked: (Recipe) -> Unit) {
  * @param userViewModel the current user view model
  */
 @Composable
-fun TopCornerUnLikeButton(recipe: Recipe, userViewModel: UserViewModel) {
+fun TopCornerUnLikeButton(
```

Comment:
It would be good to rename this composable to something like `LikeRecipeButtons` or something more general with the added download logic

Reply:
I renamed it to `TopCornerDownloadAndLikeButton`.